ios/brave-ios/Sources/Brave/Frontend/Browser/Tab.swift

```diff
@@ -429,6 +430,8 @@ class Tab: NSObject {
 
   /// Boolean tracking custom url-scheme alert presented
   var isExternalAppAlertPresented = false
+  var externalAppPopup: AlertPopupView?
```

Comment:
and why we store a `externalAppPopup`? it's not being used anywhere

Reply:
It is. We call `externalAppPopup.dismiss`